int32 tx
int32 ty
int32 tz
float32 roll
float32 pitch
float32 yaw
---
